meta/kwoptarg.rb

```diff
@@ -1,11 +1,11 @@
 Mutest::Meta::Example.add :kwarg do
-  source 'def foo(bar: baz); end'
+  source 'def foo(bar: {}); end'
```

Comment:
👍 since it seems harmful in this case. In the future lets just add a new example though. It is better than potentially introducing mutations